README.md

```diff
@@ -69,240 +63,6 @@ make test
 kubectl logs -f -n kcp-system $(kubectl get po -n kcp-system -l 'app=kmc-dev' -oname) kmc-dev
```

Comment:
In the section above, add 2 blank spaces to each line that should be indented in the unordered list.

- Run a deployment...
  >**NOTE:**...
  ```
  ko apply...
  ```

- Run tests:
  ```
  make tests
  ````

etc

Reply:
done